src/runtime_src/tools/xclbinutil/SectionBMC.cxx

```diff
@@ -36,17 +36,55 @@ SectionBMC::init::init()
 {
   auto sectionInfo = std::make_unique<SectionInfo>(BMC, "BMC", boost::factory<SectionBMC*>()); 
   sectionInfo->supportsSubSections = true;
+  sectionInfo->subSections.push_back(getSubSectionName(SubSection::FW));
+  sectionInfo->subSections.push_back(getSubSectionName(SubSection::METADATA));
+
+  // Add format support empty (no support)
+  // The top-level section doesn't support any add syntax.
+  // Must use sub-sections
+
+  sectionInfo->supportedAddFormats.push_back(FormatType::RAW);
 
   addSectionType(std::move(sectionInfo));
 }
-// -------------------------------------------------------------------------
 
-bool 
-SectionBMC::doesSupportAddFormatType(FormatType _eFormatType) const
+using SubSectionTableCollection = std::vector<std::pair<std::string, enum SectionBMC::SubSection>>;
+static const SubSectionTableCollection & 
+  getSubSectionTable() 
 {
-  // The BMC top-level section doesn't support any add syntax.  
-  // Must use sub-sections
-  return false;
+  static SubSectionTableCollection subSectionTable = {
```

Comment:
```suggestion
  static const SubSectionTableCollection subSectionTable = {
```

Reply:
While not difficult to address, I would like to instead resolve this issue with the next pull request.   Mainly because, the number of changes in this release is rather large and I want to get these long overdue changes repository sooner than later.